app/src/pages/inside/dashboardItemPage/widgetsGrid/widgetsGrid.jsx

```diff
@@ -170,7 +173,16 @@ export class WidgetsGrid extends Component {
             <div
               key={widgetId}
               className={cx('widget-view')}
-              data-grid={{ x, y, w, h, minW: 4, minH: 4, i: String(widgetId) }}
+              data-grid={{
+                x,
+                y,
+                w,
+                h,
+                minW: 4,
+                minH: 4,
+                i: String(widgetId),
+                static: this.isStaticWidget(widgetType),
```

Comment:
The `static` option is also disable the move widget functionality, I think that it is unexpected (see the screenshot below).
![image](https://user-images.githubusercontent.com/31861805/61631319-bac22880-ac92-11e9-8736-a94f20ac2a03.png)

Reply:
fixed